app/src/main/java/org/schabi/newpipe/player/PlayQueueActivity.java

```diff
@@ -280,46 +277,8 @@ private void buildControls() {
     }
 
     private void buildItemPopupMenu(final PlayQueueItem item, final View view) {
```

Comment:
I'd remove these two methods (`buildItemPopupMenu` and `heldQueueItem`) that do nothing but call another method. Just put `openPopupMenu` directly.